core/src/test/java/org/elasticsearch/index/translog/CountedBitSetTests.java

```diff
@@ -79,6 +82,7 @@ public void testReleaseInternalBitSet() {
             assertThat(countedBitSet.isInternalBitsetReleased(), equalTo(true));
             assertThat(countedBitSet.length(), equalTo(numBits));
             assertThat(countedBitSet.cardinality(), equalTo(numBits));
+            assertThat(countedBitSet.ramBytesUsed(), lessThan(ramBytesUsedWithBitSet));
```

Comment:
Make the constant package private and assert that the size here is `BASE_RAM_BYTES_USED`?

Reply:
I pushed https://github.com/elastic/elasticsearch/pull/27632/commits/8dbf71dedccbaa0878cb46b3b0782d9af846abb9